lib/pbench/server/hierarchy.py

```diff
@@ -268,13 +267,19 @@ def add_verify_list(self, controller):
             self.verifylist.append(controller)
 
     def check_controller_in_controller(self):
-        return any(
+        check = any(
             [self.validation_list[val].getlist() for val in self.validation_list]
         )
+        return check or self.bad_controllers
```

Comment:
Hmm; could perhaps be a little more efficient as return self.bad_controllers or any(...) but that's probably a really minor point as I'd assume bad_controllers would be a rare condition.

Reply:
ok